presto-cassandra/pom.xml

```diff
@@ -13,12 +13,19 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
-        <cassandra.version>2.1.0</cassandra.version>
+        <cassandra.version>2.1.3</cassandra.version>
         <datastax.version>2.1.5</datastax.version>
     </properties>
 
     <dependencies>
         <dependency>
+            <groupId>org.slf4j</groupId>
```

Comment:
Please, don't use slf4j for logging. Either use java.util.logging or airlift's lightweight wrapper (https://github.com/airlift/airlift/blob/master/log/src/main/java/io/airlift/log/Logger.java)